### PR TITLE
This PR will make MD linter happy

### PR DESCRIPTION
**Pull request by Neftedollar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +20,7 @@ Fornax is a static site generator using type safe F# DSL to define page template
 
 ## Installation
 
-Fornax is released as a global .Net Core tool. You can install it with `dotnet tool install fornax -g`
+Fornax is released as a global .Net Core tool. You can instVall it with `dotnet tool install fornax -g`
```
Comment (FoggyFinder) [inline README.md]: Is it a typo?

Comment (Neftedollar) [reply README.md]: yes, thnx

Comment (Neftedollar) [reply README.md]: fixed

**Comment by Krzysztof-Cieslak** [comment]
Thanks!

**Pull request merged**